scalafmt-tests/src/test/scala/org/scalafmt/cli/CliTest.scala

```diff
@@ -326,10 +331,15 @@ class CliTest extends AbstractCliTest {
     noArgTest(
       string2dir(
         """|/foo.scala
+           |object    FormatMe
+           |/foo.sc
            |object    FormatMe
            |""".stripMargin
       ),
-      """|/foo.scala
+      """|/foo.sc
+         |object FormatMe
```

Comment:
Note that I think package clauses are rejected by the sbt parser, not sure how problematic that is for .sc files.

We'd need to fix the scalameta parser to handle package with sbt dialect.